rpm/build/SOURCES/jenkins.init.in

```diff
@@ -112,7 +112,7 @@ RETVAL=0
 case "$1" in
     start)
 	echo -n "Starting @@PRODUCTNAME@@ "
-	daemon --user "$JENKINS_USER" --pidfile "$JENKINS_PID_FILE" "$JAVA_CMD" $PARAMS > /dev/null
+	daemonize -u "$JENKINS_USER" -p "$JENKINS_PID_FILE" $JAVA_CMD $PARAMS
```

Comment:
Why did you replace long args with short args here?

Reply:
And why stop quoting java_cmd?

Reply:
Looks like `daemonize` doesn't have long opts: http://software.clapper.org/daemonize/daemonize.html

And the command and its args are expected as additional args, not just one.